CHANGELOG.md

```diff
@@ -1,3 +1,57 @@
+# 1.7.0-beta02 (September 2024)
+
+_Changes since release/1.7.0-beta01_
+
+## Breaking changes
+### Desktop
+- [`Modifier.onExternalDrag` has been deprecated in favor of the new `Modifier.dragAndDropTarget`](https://github.com/JetBrains/compose-multiplatform-core/pull/1528)
+
+### Resources
+- [Deprecate resources in `compose.ui` in favour of the new resource library](https://github.com/JetBrains/compose-multiplatform-core/pull/1457)
+
+## Features
+### Multiple Platforms
+- [`material3-adaptive-navigation-suite` is multiplatform now](https://github.com/JetBrains/compose-multiplatform-core/pull/1539)
+
+### iOS
+- [Added support for input methods (languages such as Chinese, Korean, Arabic) to BasicTextField(TextFieldState, ...)](https://github.com/JetBrains/compose-multiplatform-core/pull/1540)
```

Comment:
Not sure about correct wording here because there are still some known issues with BTF2

Reply:
I did the same notes as desktop. Maybe basic support would be better then?

Reply:
But for desktop it was a fix with a real issue with such languages. It looks this wording is not correct for iOS at all